### PR TITLE
github-electron: update webContents module interface

### PR DESCRIPTION
**Pull request by miniak**

### CODE REVIEW

File: github-electron/github-electron.d.ts [hunk 1]
```diff
@@ -822,6 +868,203 @@ declare namespace Electron {
...
+		 * Emitted when debugging session is terminated. This happens either when
+		 * webContents is closed or devtools is invoked for the attached webContents.
+		 */
+		on(event: 'detach', listener: (event: any, reason: string) => void): this;
+		/**
+		 * Emitted whenever debugging target issues instrumentation event.
+		 * Event parameters defined by the ‘parameters’ attribute in the remote debugging protocol.
+		 */
+		on(event: 'message', listener: (event: any, method: string, params: any) => void): this;
+		on(event: string, listener: Function): this;
```
Comment (rhysd) [inline github-electron/github-electron.d.ts]: Did you forget removing this line?

Comment (miniak) [inline github-electron/github-electron.d.ts]: Unfortunatelly, this is a pimitation of TypeScript, the generic case is required :(

**Comment by miniak** [comment]
The generic event handler is required by TypeScript, unfortunatelly

**Comment by rhysd** [comment]
Yes, as I agreed in #8648 ...

:+1:

**Comment by jedmao** [comment]
:+1:

**Comment by miniak** [comment]
Superseded by https://github.com/DefinitelyTyped/DefinitelyTyped/pull/8723

**Pull request closed without merging**